pkg/specgen/generate/security.go

```diff
@@ -178,6 +178,10 @@ func securityConfigureGenerator(s *specgen.SpecGenerator, g *generate.Generator,
 
 	g.SetRootReadonly(s.ReadOnlyFilesystem)
 
+	noUseIPC := s.IpcNS.NSMode == specgen.FromContainer || s.IpcNS.NSMode == specgen.FromPod || s.IpcNS.NSMode == specgen.Host
```

Comment:
Why are you tying IPC to Network namespace?

Reply:
I don't think I am? These are all `s.IpcNS`? We're checking for `--ipc=container:` and pod IPC.

Reply:
Ok I read it wrong.